generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -435,10 +435,18 @@
         var trackItemWidth = 420;
         var currentIndex = 0;
 
-        emailInput.addEventListener("focus", function() {
+        emailInput.addEventListener("click", function() {
           emailInput.classList.remove("invalid");
         });
-        function onNext(e) {
+
+        document.addEventListener("keydown", function(e) {
```

Comment:
Shouldn't this get binded to `snippet` and not document to avoid having `onNext()` when user is actually using the search bar?